web/themes/custom/usagov/usagov.theme

```diff
@@ -295,9 +295,15 @@ function usagov_preprocess_block(&$variables) {
   /**
    * Pass node or term value to pages so they can access properties
    */
-  $entity = \Drupal::routeMatch()->getParameter('node') ?? \Drupal::routeMatch()
-    ->getParameter('taxonomy_term');
-  $variables['entity'] = $entity;
+  if (\Drupal::routeMatch()->getParameter('node')) {
+    $entity = \Drupal::routeMatch()->getParameter('node');
+    $variables['node'] = $entity;
+  }
+  elseif (\Drupal::routeMatch()
+    ->getParameter('taxonomy_term')) {
+    $entity = \Drupal::routeMatch()
```

Comment:
Do you need a `$variables['entity'] = $entity` to pass the term to twig templates?

Reply:
Good question but for now not needed. Might come up later though as the new wizard mobile nav still needs to be built.

Reply:
Added a return statement as I was seeing php errors on user pages.